model/job_test.go

```diff
@@ -222,3 +223,36 @@ func TestJobsSort(t *testing.T) {
 	assert.Equal(jobs[0].Name, "ccc")
 	assert.Equal(jobs[1].Name, "ddd")
 }
+
+func TestJobsProperties(t *testing.T) {
```

Comment:
Were all the old tests unnecessary? It seems like you're covering a lot fewer of the edge cases here.